### ISSUE TITLE
Audio & Streaming Switches are not Suppressed when set to false

### ISSUE THREAD

**Issue by mickeyc515** [Brandawg93/homebridge-nest-cam#173]
**Describe the bug**
Audio & Streaming switches are not suppressed when their flags are set to false. 

**To Reproduce**

1. Set flags to true
2. Restarted
3. Set flags to false
4. Restarted

**Expected behavior**
Suppression of the switches when set to false

Please answer these questions before submitting this issue:
```
On what OS are you running homebridge: 1.1.1
On what iOS version are you having issues: iOS 13.6
What camera types are having issues: nest cam outdoor
Does this issue occur consistently or randomly: consistently after v4.4.1 (or that’s when I noticed it)
If applicable, what was the last version that worked properly: 4.4 i believe
```

**Comment by Brandawg93** [comment]
This should be fixed in v4.4.2. If not, feel free to reopen this. Thanks for the bug catch!

**Comment by mickeyc515** [comment]
Fixed! Thanks for all the great work!
